apps/sms/tools/.bowerrc

```diff
@@ -0,0 +1,3 @@
+{
+  "directory": "../bower_components"
+}
```

Comment:
mmm I think you need this if you run the script direcrtly from the "tools" directory; but if you stay in `apps/sms` and run "tools/create_bridge_lib.sh" I think it works fine without this .bowerrc.

I'll think of a way to move to the SMS directory if we're not there, I did that in the past too, but don't need to do it now.